
uint32 width
uint32 height
float32 resolution
float64 map_origin_pos_x
float64 map_origin_pos_y

-----------------

float64 pos_x_robot
float64 pos_y_robot
float64 robot_a

